main.js

```diff
@@ -0,0 +1,117 @@
+document.getElementById("search").addEventListener("click", function () {
+  const userSearch = document.getElementById("search-query").value;
+  console.log(userSearch);
+
+  if (userSearch === "") {
+    alert("Please enter the name of a city");
+  } else {
+    fetchWeatherData(userSearch);
+  }
+
+  document.getElementById("search-query").value = "";
+});
+
+const fetchWeatherData = (city) => {
+  const url = `https://api.openweathermap.org/data/2.5/forecast?q=${city}&appid=a01dd8f2b7b0fd48756b05d7cb1e2fe5`;
+  const searchButton = document.getElementById("search");
+  searchButton.disabled = true;
+  searchButton.innerHTML = `<i class="fa fa-spinner fa-spin"></i>Loading
+`;
+  fetch(url, {
+    method: "GET",
+    dataType: "json",
+  })
+    .then((data) => data.json())
+    .then((data) => {
+      displayWeatherData(data);
+      searchButton.disabled = false;
+      searchButton.innerHTML = "Search";
+    })
+    .catch(handleError);
+};
+
+fetchWeatherData("Charlotte");
+
+const displayCurrentWeather = (data) => {
+  document.querySelector(".current-weather").replaceChildren();
+  const fahrenheitTemp = convertKelvinToFahrenheit(data.list[0].main.temp);
+  const iconURL = `https://openweathermap.org/img/wn/${data.list[0].weather[0].icon}@2x.png`;
+
+  const template = ` 
+    <div class="col-md-6 justify-center">
+      <h2>${fahrenheitTemp}&deg</h2>
+      <h3 id="city">${data.city.name}</h3>
+      <h4 id="current-condition">${data.list[0].weather[0].main}</h4>
+    </div>
+    <div class="col-md-6">
+      <img
+        src= "${iconURL}"
+        />
+      </div>`;
+
+  document
+    .querySelector(".current-weather")
+    .insertAdjacentHTML("beforeend", template);
+};
+
+const displayForecast = (data) => {
+  document.querySelector(".forecast").replaceChildren();
+  const dayOne = data.list[7];
+  const dayTwo = data.list[15];
+  const dayThree = data.list[23];
+  const dayFour = data.list[31];
+  const dayFive = data.list[39];
+  const forecastDisplay = [dayOne, dayTwo, dayThree, dayFour, dayFive];
```

Comment:
DRY, can you think of a more efficient (less lines of code) to do this using an array helped method? 
hint, map :)